ChebsNecromancy/Items/OrbOfBeckoning.cs

```diff
@@ -263,7 +263,7 @@ private void SpawnSkeleton()
 
             // if players have decided to foolishly restrict their power and
             // create a *cough* LIMIT *spits*... check that here
-            if (SkeletonWand.MaxSkeletons.Value > 0)
+            if (SkeletonMinion.MaxSkeletons.Value > 0)
```

Comment:
MaxSkeletons does not represent the actual limit

Reply:
In this case it is ok because the actual limit, including bonus minions, is determined within `SkeletonMinion.CountActiveSkeletonMinions`. So this particular if statement is a little misleading as to its real purpose. I will refactor for readability

Reply:
The limit including bonus minions is stored in `maxMinions`, which is not public, nor does it persist. The config option `MaxMinions` is public but should probably be private. There should probably be a public static field for the max number of spooky scary skeletons. Or for the skeleton slots remaining (max skellies - current skellies).

Reply:
Oh, I see what you're saying. I was thinking it would be possible for someone to have this set to 0, but gain skellies as their Necromancy increases. That might be a "but why?" situation, though, and this isn't the issue I thought it was.

Reply:
> The limit including bonus minions is stored in `maxMinions`, which is not public, nor does it persist. 

This is ok because it should be calculated dynamically based off the config value `MaxMinions` and the added bonus minions from the necromancy level. It's expected that either one will change (config update from server or player levelling up) so it needn't persist.

> The config option `MaxMinions` is public but should probably be private.

The `SkeletonMinion.MaxSkeletons` and `SkeletonMinion.MaxDraugr` config options are public because they're read by items eg. Skeleton Wand, Draugr Wand, Orb of Beckoning. That said, Rider is constantly nagging me about things that could be restricted in scope so I might open a refactoring branch later to squash all such warnings/recommendations.

Reply:
Good Rider! 😁